spec/requests/quizzes_spec.rb

```diff
@@ -0,0 +1,97 @@
+require "rails_helper"
+
+RSpec.describe "Quizzes", type: :request do
+  describe "GET index" do
+    before do
+      Quiz.create(title: "First Quiz", description: "This is the first quiz")
+      Quiz.create(title: "Second Quiz", description: "This is the second quiz")
+      Quiz.create(title: "Third Quiz", description: "This is the third quiz")
+      Quiz.create(title: "Fourth Quiz", description: "This is the fourth quiz")
+    end
+
+    it "gets all quizzes and checks the content" do
+        get "/quizzes"
```

Comment:
```suggestion
        get quizzes_path
```

Let's use the helper method here for the path - you use a helper method later in this spec so it is good to be consistent.